### DISCUSSION TITLE
Skipflag for save prompt (by DrippingYellow)

### DISCUSSION THREAD

**Discussion by TrashboxBobylev** [DoukutsuRandamu/IssueTracker#74] [Suggestions]
I bet most people appreciate the reminder that the save file only lasts until you die, but I feel like they only need to hear that once. As-is, the player needs to mash through at least 4 lines of dialogue before being able to save, which isn't much but is still a minor annoyance.

**Comment by Fluff8836** [comment]
In order to satisfy DrippingYellow's request it is clearly not going to be a skipflag but to make use of Flag 431 from vanilla Cave Story. I suggest you add a Flag Jump in Event 16 to a new event in Head.tsc.

You could create Event 2 in Head.tsc that shows only one line of dialog for save prompt. Add `<FLJ0431:0002` directly under `#0016`.

Head.tsc
```
#0002
<PRI<MSG
Do you want to save?<YNJ0000<FL+0431<SVP<CLRGame saved.<NOD<END
```
```
#0016
<FLJ0431:0002
<PRI<MSG
Do you want to save?<NOD The death
will be still permament;<NOD
this is only for
breaks and multisessions.<YNJ0000<FL+0431<SVP<CLRGame saved.<NOD<END
```

**Comment by TrashboxBobylev** [comment]
Added in 1.44.2.
